main/FatesHistoryInterfaceMod.F90

```diff
@@ -358,7 +358,7 @@ module FatesHistoryInterfaceMod
 
   integer :: ih_primaryland_fusion_error_si
 
-  integer :: ih_area_si_landuse
+  integer :: ih_fracarea_si_landuse
```

Comment:
I renamed a bunch of `area` variables to be `fracarea` instead, as "area" implies m2 but most of these are fractional. There are still some `area` variables in other subroutines, like `ih_area_plant_si`, that should probably also be changed for consistency.

Reply:
thanks

Reply:
I've now changed other such variables throughout this module to match.